account_entries_report_hooks/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+===============================
+Account Entries Report Hooks
+===============================
+
+This module extends the functionality of account.entries.report module
+which intends to override it in order to support and allow you to extend
+it by other modules that want to add or change the standard behaviour.
+
+Installation
+============
+
+No external library is used.
+
+Configuration
+=============
+
+To configure this module, you need to:
```

Comment:
Remove this line. Substitute by: "In order to extend the Account Entries Report, you need to:
*  Include this module as dependency in your own module
*  Inherit from "account.entries.report" module
* Add your own fields, as needed
* Implement one of the new methods _select, _from, _where, _group_by, calling first to the super() method and then adding the extra string as needed.